app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here is a list of all posts from user 1 and bio</h1>
\ No newline at end of file
+<article class="main-article">
+  <div class="user-info-wrapper">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
```

Comment:
- [ ] [Optional] I noticed that you are repeating the same code with the same style in the `User#index` and `User#show` pages. Repeating code goes against the **DRY (Don't Repeat Yourself)** principle. To address this, I recommend using [Rails partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) to reduce code duplication. By creating partials, you can encapsulate common code and include it in both views, promoting cleaner and more maintainable code. Please consider updating your code to improve its structure and adhere to best practices. Keep up the good work! 🚀🧹